lib/rubocop/processed_source.rb

```diff
@@ -13,7 +13,12 @@ class ProcessedSource
                 :parser_error, :raw_source
 
     def self.from_file(path)
-      new(File.read(path), path)
+      begin
+        file = File.read(path)
```

Comment:
btw, you can move `new(file, path)` here and this will remove the need for the explicit `begin` block (you can use the implicit `begin` of the method then).